src/pages/regulatory/_document_accordion.tsx

```diff
@@ -164,7 +164,7 @@ const DocumentAccordion = (locale: DocumentAccordionProps) => {
                 </Text>
                 <Flex>
                     <EdgeFlex mt="1.8rem">
-                        {kid_data.map((data, idx) => (
+                        {(is_eu_country ? kid_data_eu : kid_data).map((data, idx) => (
```

Comment:
Hi @pavel-latyshou-deriv , let's store this in a variable and later do the mapping
(is_eu_country ? kid_data_eu : kid_data)